audio_switch.c

```diff
@@ -112,8 +112,10 @@ int runAudioSwitch(int argc, const char * argv[]) {
 					typeRequested = kAudioTypeInput;
 				} else if (strcmp(optarg, "output") == 0) {
 					typeRequested = kAudioTypeOutput;
-				} else if (strcmp(optarg, "system") == 0) {
-					typeRequested = kAudioTypeSystemOutput;
+                } else if (strcmp(optarg, "system") == 0) {
```

Comment:
@gileswells you should probably fix the indention here to match the original code